### PR TITLE
feat(import): DRAFT - Create a bulk import for structures

### PR DESCRIPTION
**Pull request by annelhote**

### CODE REVIEW

**Comment by annelhote** [comment]
Here is the curl command that I use to test my new endpoint in localhost : `curl --request POST \
  --url http://localhost:3000/import \
  --header 'Authorization: Bearer MY_BEARER' \
  --header 'Content-Type: multipart/form-data' \
  --form file=@/home/alhote/Downloads/AjoutEnMasseStructureTest.csv`

**Comment by annelhote** [comment]
And the file "AjoutEnMasseStructureTest.csv" content is :
```
Nom usuel en français;Nom court en français;Sigle en français;Nom long en anglais :;Nom court  en anglais :;Nom officiel dans la langue originale;Nom court officiel dans la langue originale;"Autres dénominations possibles séparées par des ;";Statut [O = Ouvert, F = Fermé, P = Potentiel];"Code de la/des catégories {rechercher le code, séparation ;}";Code du statut juridique {rechercher le code};Localisation [A1 = France métropolitaine et les DOM, A2 = Collectivités d'outre-mer, A3 = Hors de France];Code commune {rechercher le code};Identifiant IdRef [123456789];Identifiant Wikidata [Q1234];Identifiant ROR [12cd5fg89];Code UAI [1234567X];Numéro siret [12345678901234];Numéro RNSR [123456789X];Numéro de l'ED [123];Site internet en français;Site internet en anglais;Compte twitter [https://twitter.com/XXX];Compte linkedIn [https://www.linkedin.com/in/XXX-XXX-123456/];Mention de distribution :;Adresse :;Lieu dit :;Boite postale :;Code postal :;Localité d'acheminement :;Nom du pays en français {rechercher le libellé ou le code ISO};Coordonnées GPS [-12.34,5.6789];Date de création {2020-07-02};Date de création approximative {O = Oui, N = Non};Date de fermeture {2020-07-02};Date de fermeture approximative {O = Oui, N = Non};Parent {rechercher le code}
Test FR Anne;Test FA;TFA;English test;ET;;;"Nom_01;nom_02";O;;;;;123458;;;;;;;;;;;;;;;;;;;;;;;
```

File: src/api/import/import.middlewares.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      closureDate: item['Date de fermeture {2020-07-02}'],
+      creationDate: item['Date de création {2020-07-02}'],
+      nameEn: item['Nom long en anglais :'],
+      officialName: item['Nom officiel dans la langue originale'],
+      otherNames: item['Autres dénominations possibles séparées par des ;'].split(';'),
+      shortName: item['Nom court en français'],
+      structureStatus: statusMapping[item['Statut [O = Ouvert, F = Fermé, P = Potentiel]']],
+      usualName: item['Nom usuel en français'],
+    };
+    req.body = structure;
```
Comment (annelhote) [inline src/api/import/import.middlewares.js:56]: @folland87 I would like to use the router to call the `/structures` endpoint here to create a new one. But no idea how to do that.

Comment (folland87) [reply src/api/import/import.middlewares.js:56]: Inside your controller, you could make a POST `${req.hostname}/structures`, i guess.
Not sure, however, that we should implement an multipart/form-data import with csv file on the server, or let the frontend do the job of transforming csv to json and implement an application/json endpoint.

Comment (annelhote) [reply src/api/import/import.middlewares.js:56]: If we implement an API endpoint, it would be useful for external services. Let's discuss about that.

**Pull request closed without merging**